CourseManager/Views/AddOrEditCourseSheet.swift

```diff
@@ -21,60 +21,82 @@ struct AddOrEditCourseSheet: View {
-            Form {
-                courseNameSection
-                courseImageNameSection
-                courseURLSection
-                gitHubURLSection
-                percentageProgressSection
-                statusPickerSection
-                if viewModel.course.status == .complete {
-                    dateCompletedCalendarSection
+            ZStack {
+                MainColors.accent.ignoresSafeArea()
+                Form {
+                    courseNameSection
+                    courseImageNameSection
+                    courseURLSection
+                    gitHubURLSection
+                    percentageProgressSection
+                    statusPickerSection
+                    if viewModel.course.status == .complete {
+                        dateCompletedCalendarSection
+                    }
+                    commentsSection
+                    submitButtonSection
+                        .disabled(isFormDisabled)
                 }
-                commentsSection
-                submitButtonSection
-                    .disabled(isFormDisabled)
-            }
-            .onAppear(perform: {
-                if !editViewIsOpened {
-                    setUpViewModelCourseInformationToEdit()
-                }
-            })
-
-            .navigationTitle("Course Information")
-            .toolbar {
-                Button {
-                    handleXMarkTapped()
-                } label: {
-                    Image(systemName: "xmark")
-                        .font(.title2)
+                .foregroundColor(.white)
+                .background(LinearGradient(gradient: Gradient(colors: [.purple, .blue]), startPoint: .top, endPoint: .bottomLeading), in: RoundedRectangle(cornerRadius: 30, style: .continuous))
+                .clipShape(RoundedRectangle(cornerRadius: 30, style: .continuous))
+                .shadow(color: AppColors.shadow.opacity(0.4), radius: 5, x: 5, y: 5)
+                .scrollContentBackground(.hidden)
+                .padding(20)
+                .onAppear(perform: {
+                    if !editViewIsOpened {
+                        setUpViewModelCourseInformationToEdit()
+                    }
+                })
+
+                .navigationTitle("Course Information")
+                .toolbar {
+                    Button {
+                        handleXMarkTapped()
+                    } label: {
+                        Image(systemName: "xmark")
+                            .font(.title2)
+                    }
                 }
+                .toolbar(.hidden, for: .tabBar)
             }
         }
     }
 
     var courseNameSection: some View {
         Section(header: Text("Course Name")) {
             TextField("Enter course name", text: $viewModel.course.name)
-        }.textFieldStyle(.roundedBorder)
+        }
+        .textFieldStyle(.plain)
+        .textFieldStyle(.roundedBorder)
+        .listRowBackground(MainColors.accent.opacity(0.7))
```

Comment:
Can you try to set `.listRowBackground(MainColors.accent.opacity(0.7))` for the `List`, because in the SwiftUI it should apply this modifier for each node of the List 

```
Form {

}.listRowBackground(MainColors.accent.opacity(0.7))
```

Reply:
This didn't work, but I instead put all in the sections in a group and set  .listRowBackground(MainColors.accent.opacity(0.7)) on the group and that worked. 
Form {
         Group {
              courseNameSection
              courseURLSection
      }.listRowBackground(MainColors.accent.opacity(0.7))
}